tests/runtests.py

```diff
@@ -162,14 +161,6 @@ def no_available_apps(self):
     log_config['loggers']['django']['level'] = 'ERROR'
     settings.LOGGING = log_config
 
-    if verbosity > 0:
-        # Ensure any warnings captured to logging are piped through a verbose
-        # logging handler.  If any -W options were passed explicitly on command
-        # line, warnings are not captured, and this has no effect.
-        logger = logging.getLogger('py.warnings')
-        handler = logging.StreamHandler()
-        logger.addHandler(handler)
-
```

Comment:
As far as I tested, this isn't required for that behavior. We still have `warnings.simplefilter("error", RemovedInDjango20Warning)` in this file for that. I'm not sure exactly what, if anything, this does.